database-migration/migrations/6_hosts_index_createdAt.go

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) 2022 Sorint.lab S.p.A.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+package migrations
+
+import (
+	"context"
+
+	migrate "github.com/xakep666/mongo-migrate"
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/mongo"
+)
+
+func init() {
+	migrate.Register(func(db *mongo.Database) error {
+		if err := create_index_hosts(db); err != nil {
```

Comment:
May you use create_index_hosts directly instead of wrap it around another func?
Like:
```
diff --git a/database-migration/migrations/6_hosts_index_createdAt.go b/database-migration/migrations/6_hosts_index_createdAt.go
index 0363578f..3a193eff 100644
--- a/database-migration/migrations/6_hosts_index_createdAt.go
+++ b/database-migration/migrations/6_hosts_index_createdAt.go
@@ -24,15 +24,10 @@ import (
 )
 
 func init() {
-	migrate.Register(func(db *mongo.Database) error {
-		if err := create_index_hosts(db); err != nil {
-			return err
-		}
-		return nil
-
-	}, func(db *mongo.Database) error {
-		return nil
-	})
+	migrate.Register(create_index_hosts,
+		func(db *mongo.Database) error {
+			return nil
+		})
 }
 
 func create_index_hosts(db *mongo.Database) error {
```